_rules/table-header-cell-has-assigned-cells-d0f69e.md

```diff
@@ -16,42 +16,47 @@ input_aspects:
   - DOM Tree
 acknowledgments:
   authors:
+    - Aron Janecki
     - Audrey Maniez
+    - Jean-Yves Moyen
     - Jey Nandakumar
   funding:
     - WAI-Tools
 ---
 
 ## Applicability
 
-This rule applies to any [HTML element][] with a [semantic][semantic role] [rowheader][] or [columnheader][] for which all of the following is true:
+This rule applies to any [non-empty][] `th` element that is [included in the accessibility tree][], has a [semantic role][] of [rowheader][] or [columnheader][], and for which all of the following are true:
 
-- the element is [visible][]; and
-- the element is [included in the accessibility tree][]; and
-- the element has at least one ancestor in the [flat tree][] that is a [semantic][semantic role] [table][] or [grid][]; and
-- the element's closest ancestor in the [flat tree][] that is a [semantic][semantic role] [table][] or [grid][] is [included in the accessibility tree][].
+- **in a table** the `th` element is a [descendant][] of a `table` element for which all of the following are true:
+  - **visible** the `table` element is [visible][]; and
+  - **minimum rows** the `table` element's [`rows` collection](https://html.spec.whatwg.org/multipage/tables.html#dom-table-rows) has 2 or more `tr` elements; and
+  - **non-empty data cell** the `table` element has at least one [non-empty][] `td` element that is a [child][] of a `tr` element with the [`rowIndex`][rowindex] value of 1 or more if the `tr` element with the value of 0 has at least two `th` [children][child] or the value of 0 or more if the `tr` element with the value of 0 has one or less `th` [children][child].
```

Comment:
```suggestion
  - **non-empty data cell** the `table` element has at least one [non-empty][] `td` element that is a [child][] of a `tr` element with the [`rowIndex`][rowindex] value of 1 or more if the `tr` element with the value of 0 has at least two `th` [children][child]; or the value of 0 or more if the `tr` element with the value of 0 has one or less `th` [children][child].
```

Interestingly, this condition would make Jean-Yves' one row example, applicable.